src/libstd/collections/hash/table.rs

```diff
@@ -1061,6 +1129,45 @@ impl<K: Clone, V: Clone> Clone for RawTable<K, V> {
     }
 }
 
+impl<K: Debug, V: Debug> Debug for RawTable<K, V> {
```

Comment:
Should we keep these out to avoid unnecessary code? The end user can't use these anyway.